internal-rules/out/report-message-format.test.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
```

Comment:
Do we really need to versioning these *.js files?

Reply:
You mean all the js files, or jusst the tests?

Reply:
I mean all... I know that we need the compiled files to lint the rules, but can't we have a smarter mechanism rather than versioning these?